store.go

```diff
@@ -515,6 +525,12 @@ func (s *store) load() error {
 	}
 	s.layerStore = rls
 
+	rols, err := s.ROLayerStores()
+	if err != nil {
+		return err
+	}
+	s.roLayerStores = rols
```

Comment:
This assignment looks redundant, since `ROLayerStores()` is already making sure `s.roLayerStores` is set.